### ISSUE TITLE
bigger learning rate leads to reach minimum loss scale

### ISSUE THREAD

**Issue by zherowolf** [pytorch/fairseq#282]
I have encountered “Exception: Minimum loss scale reached ” error several times with fp16 training.
That's because the loss scale using in fp16 training reached the min_loss_scale (0.0001 default).
Sometimes my loss explodes (with learning rate 0.001),  but sometimes the loss seems ok.
So my question is : can I set hp min_loss_scale to zero to avoid exceptions when the loss is ok?
and what learning rate do you usually use to accelerate training (2x lr in the paper)?
![image](https://user-images.githubusercontent.com/41291189/45815226-9613c200-bd0a-11e8-967f-ff7915e8dcee.png)
(log without personal info)

**Comment by pradnyadesai** [comment]
Hi,
I'm currently training the Google Speech Commands dataset using the code in the link provided in this paper (https://arxiv.org/pdf/1904.05862v3.pdf).
Here is the code: 
![image](https://user-images.githubusercontent.com/50212402/66362466-e5965180-e950-11e9-9a7e-b85f6c64299d.png)
I tried running this code on a fraction of the dataset and it ran fine for over a 1000 epochs. I then deleted the checkpoints and tried running the code on the entire dataset. When I do this, it eventually gives me this error:
![image](https://user-images.githubusercontent.com/50212402/66363805-7b33e000-e955-11e9-82c2-62461e6ed394.png)
I tried lowering the learning rate, tried gradient clipping, and also tried increasing the batch size. Nothing seems to work. Does anyone know how I can resolve this?
